File: Practice/21/21/21.cpp
#include <iostream>
using namespace std;

double BMI(double weight, double height)
{
	double BMI = weight / (height * height);
	return BMI;
}

void printBMI(double BMI)
{
	if (BMI < 18.5)
		cout << "Underweight";
	else if (BMI < 25.0)
		cout << "Normal weight";
	else if (BMI < 30.0)
		cout << "Overweight";
	else
		cout << "Obesity";
}

int main()
{
	double weight, height;
	cin >> weight >> height;
	height = height / 100;
	printBMI(BMI(weight, height));
}

File: Practice/21/Python/21.py
def bmi(weight: float, height: float) -> float:
    return weight/(height**2)

def print_bmi(bmi: float) -> float:
    if bmi < 18.5:
        print("Underweight")
    elif bmi < 25.0:
        print("Normal weight")
    elif bmi < 30.0:
        print("Overweight")
    else:
        print("Obesity")

weight, height = map(float, input().split())
height /= 100
print_bmi(bmi(weight, height))
